SingleCellPatch/extract_patches.py

```diff
@@ -133,102 +136,25 @@ def generate_mask(positions, positions_labels, cell_id, window, window_segmentat
         target_mask.reshape((x_size, y_size, 1)), \
         target_mask2.reshape((x_size, y_size, 1))
 
-
-def instance_clustering(cell_segmentation, 
-                        ct_thr=(500, 12000), 
-                        instance_map=True, 
-                        map_path=None, 
-                        fg_thr=0.3,
-                        DBSCAN_thr=(10, 250)):
-    """ Perform instance clustering on a static frame
-
+def check_segmentation_dim(segmentation):
```

Comment:
This function would not behave properly after changing the dimension order. You might want to change/remove it.